Dockerfile_pypy

```diff
@@ -13,8 +13,7 @@ RUN pypy3 -m pip install scipy
 
 # TODO: Workaround for pysolc due to https://github.com/ethereum/py-solc/issues/64
```

Comment:
Probably this line could be deleted too if that [line](https://github.com/gridsingularity/d3a/pull/627/files#diff-43324f89e338058c3de62665c5d1802dL17) is being removed.

Reply:
You are right, deleting this. Thanks